core/common/connector-core/build.gradle.kts

```diff
@@ -36,7 +36,8 @@ dependencies {
     implementation(libs.dnsOverHttps)
     implementation(libs.bouncyCastle.bcpkixJdk18on)
     implementation(libs.nimbus.jwt)
-    
+    implementation(libs.tink)
```

Comment:
is this needed?

Reply:
yes, because there is no implementation for EdDSA Keys in Nimbus, cf. https://connect2id.com/products/nimbus-jose-jwt#deps